api/src/main/java/jakarta/json/spi/JsonProvider.java

```diff
@@ -66,6 +84,10 @@ protected JsonProvider() {
      * @return a JSON provider
      */
     public static JsonProvider provider() {
+        String factory = System.getProperty(JsonProvider.class.getName());
```

Comment:
the main point of this change is javadoc update. It must contain exact description of the lookup algorithm, so step 1, step 2, etc; sth like _Discovery..._ in [JAXBContext](https://jakarta.ee/specifications/xml-binding/3.0/apidocs/jakarta.xml.bind/jakarta/xml/bind/jaxbcontext)

Reply:
Sorry, just noticed already approved.  Above comment is not critical.

Reply:
addressed in #333